tags.go

```diff
@@ -0,0 +1,42 @@
+package distribution
+
+// Tag is a named reference to a repository object. Target generally refers to
+// a manifest.
+type Tag interface {
+	Blob // io access to serialized content.
+
+	// Name returns the fully qualified name of the object. For example, if
+	// the object is in repository "stevvooe/test" and has tag "latest", the
+	// name will be "stevvooe/test:latest".
+	Name() string
+
+	// Description returns a human-readable description of the tag.
```

Comment:
name: "stevvooe/test:latest"
description: "stevvooe/test:latest is an example used to demonstrate the return value of the `Name` method on the `Tag` interface. It is not a real image name. This description demonstrates the value of a human readable description that may help to identify the semantic importance of the tag. It is debatable whether or not this field is necessary."